### PR TITLE
Azure rm upgrade changes

### PR DESCRIPTION
**Pull request by kumarmadurai**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by amitchidrewar1301 — DISMISSED** [review]
Changes are fine for me

File: infrastructure/main.tf [hunk 1]
```diff
@@ -1,3 +1,4 @@
 provider "azurerm" {
-  version = "1.44.0"
+  version = "=2.20.0"
```
Comment (amitchidrewar1301) [inline infrastructure/main.tf:2]: As discussed please check if we need to add empty perfest.tfvars . If the Answer is yes please add tfvars for all the environment as well, same checks need to be applied for fees-register-admin-web, bar-admin-web.

Comment (kumarmadurai) [reply infrastructure/main.tf:2]: I triggered build in perftest without perftest.tfvers file. It's working fine, hence I'm removing perftest.tfvers file from all PRS(pay bubble, fees-register-admin-web, bar-admin-web)

Comment (amitchidrewar1301) [reply infrastructure/main.tf:2]: PR looks good to me

**Review by amitchidrewar1301 — APPROVED** [review]

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**